Business & Real-World/README.md

```diff
@@ -15,9 +15,13 @@
 ## Conference Schedule
 
 ### Toolbox
-* [faker.js](https://github.com/Marak/Faker.js) (github.com)
+* [faker.js](https://fakerjs.dev/) (fakerjs.dev)
 
 ### Tutorials
+* [create-massive-random-data-using-faker-js](https://medium.com/nerd-for-tech/create-massive-random-data-using-faker-js-f47909f6fab5) (medium.com)
+* [Getting Started With Faker.js: A Developer’s Guide](https://www.testim.io/blog/getting-started-with-faker-js/) (testim.io)
+* [Getting Started](https://fakerjs.dev/guide/) (fakerjs.dev)
```

Comment:
we could probably leave this one off since we're already linking to the library itself above

Reply:
@colbyfayock  did you mean only leaving  [Getting Started](https://fakerjs.dev/guide/) (fakerjs.dev) out or all the tutorial including the youtube video. Sorry I am bit confused 🙂

Reply:
oh i only meant the fakerjs.dev guide, you could have left the others, ill wait to merge in case youw ant to add some back. your call!

Reply:
Sorry , I will  add them quickly now 🙂